src/bp/ui-shared/src/ConfirmDialog/index.tsx

```diff
@@ -41,6 +41,7 @@ const ConfirmDialogComponent: FC<ConfirmDialogProps> = props => {
           id="confirm-dialog-accept"
           className={Classes.BUTTON}
           type="button"
+          autoFocus
```

Comment:
Let me tap this again. This should not be part of this PR ;)

Reply:
Agreed, but it's small so I'll leave it 😬  it was a big pain not to have it when testing the empty state for qna